src/compiler/config/validate-service-worker.ts

```diff
@@ -17,9 +17,7 @@ export const validateServiceWorker = (config: d.ValidatedConfig, outputTarget: d
     return;
   }
 
-  if (outputTarget.serviceWorker === true) {
```

Comment:
Feel free to push back on this.

Reply:
You know, after the last round of fixes, I wonder if we should just go ahead and restore this in a similar fashion to https://github.com/ionic-team/stencil/pull/3863/files#diff-d9e941746821b7e6229ddee0bdbf0fca566b115a9e6b75bc4d3bcaa88c598443R20 - that's where I lean currently (and add tests!). Thoughts?

Reply:
Err NVM - we have tests for this already. This works 👍